WebApiCoreSeed.UnitTests/Services/BaseServiceUnitTest.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Runtime.CompilerServices;
+using WebApiCoreSeed.Data.EF;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using WebApiCoreSeed.Data.Models;
+using System;
+
+namespace WebApiCoreSeed.UnitTests.Services
+{
+    public class BaseServiceUnitTest
+    {
+        public WebApiCoreSeedContext Context { get; set; }
+
+        public User User { get; set; }
+
+        [TestInitialize]

```

Comment:
I'm not too familiar with xUnit, however I think it's a good idea to keep it native whether possible. Besides reasons for "not to" for MSTest, I would like to learn what'd be the advantages of using xUnit though. Thanks!